config/config.yaml

```diff
@@ -210,6 +210,8 @@ tide:
   # Default tide config for all repos in the cert-manager org
   - orgs:
     - cert-manager
+    excludedRepos:
+    - cert-manager/cert-manager # Handled with a separate Tide query below.
```

Comment:
This should fix tide for cert-manager/cert-manager, where it was running two queries instead of one.